regolith/helpers/l_projectahelper.py

```diff
@@ -140,32 +144,24 @@ def sout(self):
                     continue
             if rc.grant and rc.grant not in projectum.get('grants'):
                 continue
-            if rc.ended and not projectum.get('end_date'):
-                continue
             if rc.ended:
-                end_date = projectum.get('end_date')
-                if isinstance(end_date, str):
-                    end_date = date_parser.parse(end_date).date()
-                low_range = desired_date - dt.timedelta(days=num_of_days)
-                high_range = desired_date + dt.timedelta(days=num_of_days)
-                if low_range <= end_date <= high_range:
-                    end_projecta.append(projectum)
+                if projectum.get('status') not in ACTIVE_STATI:
+                    if projectum.get('status') != 'finished' or projectum.get('end_date') == None or type(projectum.get('end_date')) != dt.date:
+                        error_projecta.append(projectum)
+                    else:
+                        end_date = projectum.get('end_date')
+                        if isinstance(end_date, str):
+                            end_date = date_parser.parse(
+                                end_date).date()
+                        low_range = desired_date - \
+                            dt.timedelta(days=num_of_days)
+                        if low_range <= end_date <= desired_date:
```

Comment:
@sbillinge 
Just realized I was missing part of the range filtering here.
Can you merge/check this before group meeting tmrw?